scripts/install.ps1

```diff
@@ -1,3 +1,7 @@
+$p = New-Object System.Security.Principal.WindowsPrincipal([System.Security.Principal.WindowsIdentity]::GetCurrent())
+if (!$p.IsInRole([System.Security.Principal.WindowsBuiltInRole]::Administrator)) {
+    throw 'This script requires admin privledges to run and the current Windows PowerShell session is not running as Administrator. Start Windows PowerShell by  using the Run as Administrator option, and then try running the script again.'
```

Comment:
@natecanfield822 this seems to work well, however could you double check the wording, for example privileges is misspell. There are also more than 1 spaces between words sometimes.